File: tests/pass/expressions.c
int add(int k, int y)
{
    return k + y;
}

int sum(int a, int b)
{
    // a + b
    /*
     * comment
     */
    int r = (a + b) / 10;
    return r;
}

int main(int K)
{
    int a = 10;
    int b = 20;
    int r = sum(a, b) + add(a, b);
    return 0;
}

File: tests/pass/expressions.lua
local function add(k, y)
	return (k + y)
end

local function sum(a, b)
	local r = ((a + b) / 10)
	return r
end

os.exit((function(args)
	local args_table = args
	local K = args_table and tonumber(args_table[1]) or 0
	local a = 10
	local b = 20
	local r = (sum(a, b) + add(a, b))
	return 0
end)(arg))
